packages/@expo/cli/e2e/__tests__/customize-test.ts

```diff
@@ -84,3 +88,39 @@ it(
   // Could take 45s depending on how fast npm installs
   120 * 1000
 );
+
+it(
+  'runs `npx expo customize tsconfig.json`',
+  async () => {
+    const projectRoot = await setupTestProjectAsync('expo-typescript', 'with-router', '48.0.0');
```

Comment:
Test that this can be run on a project that already has typescript setup and that routes will be generated. The tsconfig should be kept intact.